projects/packages/my-jetpack/src/products/class-search-stats.php

```diff
@@ -0,0 +1,89 @@
+<?php
+/**
+ * Get search stats for use in the wp-admin dashboard.
+ *
+ * @package automattic/jetpack-search
```

Comment:
Do we want to update this to `@package automattic/my-jetpack`?

Reply:
yes, we do. Thanks